pyproject.toml

```
@@ -23,10 +23,10 @@ classifiers = [
     "Intended Audience :: Developers",
 ]
 dependencies = [
-    "typer>=0.7.0",
-    "ghga-service-commons[api]>=0.4.3",
-    "ghga-event-schemas==0.13.4",
-    "hexkit[mongodb,s3,akafka]>=0.10.2",
+    "ghga-service-commons[all]>=1.0.1",
```

```suggestion
    "ghga-service-commons[all]>=1.1.0",
```
1.1.0 contains the multi node object storage config

---

Done